clang-tools-extra/test/clang-tidy/checkers/readability/use-explicit-namespaces.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %check_clang_tidy %s readability-use-explicit-namespaces %t
+
```

Comment:
Add tests with:
- templates
- operators
- classes
- fixes
- ...

Reply:
Are there tools or resources for this?  I locally have done a lot more testing, but it is very clear how to turn this into test cases.